Modules/Promotion/Http/Controllers/PromotionController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Modules\Promotion\Http\Controllers;
+
+use Illuminate\Contracts\Support\Renderable;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Modules\Promotion\Http\Requests\StorePromotionRequest;
+use Modules\Promotion\Http\Services\PromotionService;
+use Modules\User\Entities\User;
+use Modules\Brand\Entities\Brand;
+use Modules\Promotion\Entities\Promotion;
+
+class PromotionController extends Controller {
+    
+    public function __construct(PromotionService $promotionService)
+    {
+        $this->promotionService = $promotionService;
+    }
+
+    /**
+     * Display a listing of the resource.
+     * @param Request $request
+     * @return mixed
+     */
+    public function index(Request $request) {
+        $response = $this->promotionService->getPromotions($request);
+        
+        return response()->json($response);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     * @param Request $request
+     * @return mixed
+     */
+    public function store(StorePromotionRequest $request) {
+        
+        $response = $this->promotionService->store($request->all());
+        
+        return response()->json($response);
+    }
+
+    /**
+     * Show the specified resource.
```

Comment:
Comments is not descriptive and contains errors, add space between description and parameter list